diff --git a/dist/client/index.js b/dist/client/index.js
index 5ac8fb84f2ad7b45cdf217234e07d08dce254932..df07aa9b0420660fdd5da41e1369708cf8229d3a 100644
--- a/dist/client/index.js
+++ b/dist/client/index.js
@@ -8,7 +8,7 @@ const composePreambleCode = ({
   baseWithOrigin = "/",
   overlayConfig
 }) => `
-import { inject } from "${baseWithOrigin}${RUNTIME_CLIENT_RUNTIME_PATH.slice(1)}";
+import { inject } from "${RUNTIME_CLIENT_RUNTIME_PATH}";
 inject({
   overlayConfig: ${JSON.stringify(overlayConfig)},
   base: "${baseWithOrigin}",
